cmake/FindOpenVX.cmake

```diff
@@ -0,0 +1,28 @@
+ocv_clear_vars(HAVE_OPENVX OPENVX_LIBS)
+
+set(OPENVX_ROOT "" CACHE PATH "OpenVX install directory")
+
+if(OPENVX_ROOT)
+  find_path(OPENVX_INCLUDE_DIR "VX/vx.h" PATHS "${OPENVX_ROOT}/include" DOC "OpenVX include path")
+  find_library(OPENVX_openvx_LIB "openvx" PATHS "${OPENVX_ROOT}/lib")
+  find_library(OPENVX_vxu_LIB "vxu" PATHS "${OPENVX_ROOT}/lib")
+  find_library(OPENVX_helper_LIB "openvx-helper" "${OPENVX_ROOT}/lib")
+  find_library(OPENVX_extras_lib_LIB "openvx-extras-lib" "${OPENVX_ROOT}/lib")
+  find_library(OPENVX_target_LIB "openvx-c_model" "${OPENVX_ROOT}/lib")
```

Comment:
I would recommend to avoid `openvx-helper`, `openvx-extras-lib` and `openvx-c_model` libraries usage. They are artifacts of sample implementation and might not be available in other implementations.